features/nanostack/sal-stack-nanostack/nanostack/ns_sha256.h

```diff
@@ -61,45 +61,45 @@ static inline void ns_sha256_clone(ns_sha256_context *dst,
 
 static inline void ns_sha256_starts(ns_sha256_context *ctx)
 {
-    mbedtls_sha256_starts(ctx, 0);
+    (void)mbedtls_sha256_starts_ret(ctx, 0);
```

Comment:
Why not check the return code? Just ignoring the return code seems like an easy thing to do now, but hides a valuable compiler warning about us not checking the return code here, which really is the case. If we determine that checking the return code here is _wrong_, then it makes sense to ignore it explicitly with a void cast, but I'm skeptical that's the case.

Reply:
@jarvte Can you comment?

Reply:
@0xc0170 Is this something we can file an issue against nanostack for, to address later, or should it be addressed now during code review?

Reply:
I talked with nanostack team and they said at this point ingoring the return value is ok. If sha fails for what ever reason, application will notice it anyway.